packages/kbn-std/src/iteration/test_helpers.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export const list = (size: number) =>
+  ' '
+    .repeat(size)
+    .split('')
+    .filter(Boolean)
+    .map((_, i) => i);
+
+export const sleep = (ms: number) => new Promise((r) => setTimeout(r, ms));
```

Comment:
the most popular utility function in Kibana will be unnecessary soon https://nodejs.org/api/timers.html#timers_timerspromises_settimeout_delay_value_options

Reply:
😱